src/python/dxpy/nextflow/nextflow_builder.py

```diff
@@ -46,6 +56,7 @@ def parse_extra_args(extra_args):
             dx_input["whats_new"] = extra_args.get("details", {}).get("whatsNew")
         return dx_input
 
+    extra_args = extra_args or {}
```

Comment:
might not needed: see https://github.com/dnanexus/dx-toolkit/blob/c3e79d533b8633bc71239172c4b03cf07d5ce33d/src/python/dxpy/scripts/dx_build_app.py#L1211C78-L1211C88

Reply:
maybe but for whatever reason a default empty dict was given in the earlier version, which is a bad practice. I set default to None which I substitute if needed.